### PR TITLE
Newsletter 257 translate in french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Review by murchandamus — COMMENTED** [review]
Just noticed a couple small things

File: _posts/fr/newsletters/2023-06-28-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  un encodage de clé publique de 64 octets qui est informatiquement indiscernable des données aléatoires.
+  Le module `ellswift` fournit des fonctions pour encoder et décoder les clés publiques dans le nouveau
+  format ainsi que des fonctions de commodité pour générer de nouvelles clés uniformément aléatoires et
+  effectuer un échange de clés Diffie-Hellman à courbe elliptique (ECDH) sur les clés encodées ellswift.
+  L'ECDH basé sur ellswift doit être utilisé pour établir des connexions pour le protocole [transport P2P
+  chiffré version 2] [topic v2 p2p transport] ([BIP324][]). {% assign timestamp="1:40:37" %}
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="6303,2701,2696,7710,2368,2367,2319,2120,2089,2077,1129" %}
+[policy series]: /en/blog/waiting-for-confirmation/
```
[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/fr/newsletters/2023-06-28-newsletter.md:216]: Agreed, I think @Copinmalin is still working on translating the policy series for this newsletter (its still in en)

[the remark this answers is not in the document]
Comment (Copinmalin) [reply _posts/fr/newsletters/2023-06-28-newsletter.md:216]: I started out hoping that others would be able to add to it, but I think I'm alone... 

I'm now back from holiday and I'm going to try to make up for lost time

File: _posts/fr/newsletters/2023-06-28-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
+---
+title: 'Bulletin hebdomadaire Bitcoin Optech #257'
```
[the remark this answers is not in the document]
Comment (murchandamus) [reply _posts/fr/newsletters/2023-06-28-newsletter.md]: Yeah, it is easy for example with `sed`:
`sed -i 's/Bulletin Bitcoin Optech/Bulletin Hebdomadaire Bitcoin Optech/g' *.md`

I’ve opened a PR here: #1222

[the remark this answers is not in the document]
Comment (Copinmalin) [reply _posts/fr/newsletters/2023-06-28-newsletter.md]: Thanks for your help,

do you ha ve a command for auto translate in french ? 😜

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin and @murchandamus !

**Pull request merged**